cob_gazebo_worlds/urdf/izs-campus/izs-campus.urdf.xacro

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro"
+       name="izs-campus">
+
+  <xacro:include filename="$(find cob_gazebo_worlds)/urdf/common.xacro" />
+  <xacro:include filename="$(find cob_gazebo_worlds)/urdf/materials.urdf.xacro" />
+
+  <xacro:macro name="izs-campus" params="name">
+
+    <gazebo>
+      <plugin name="environment_gazebo_ros_control" filename="libgazebo_ros_control.so">
+        <robotNamespace>world</robotNamespace>
+        <robotParam>/world_description</robotParam>
+      </plugin>
+    </gazebo>
+
+    <link name="world"/>
+
+    <link name="projected_map">
+      <xacro:box_inertial x="0.0" y="0.0" z="0.0" mass="5.0">
+        <origin xyz="0 0 0" rpy="0 0 0" />
+      </xacro:box_inertial>
+      <collision name="collision">
+        <pose>0 0 0 0 0 0</pose>
+        <geometry>
+            <mesh filename="package://cob_gazebo_worlds/Media/models/izs-campus/meshes/map.stl" />
+        </geometry>
+      </collision>
+      <visual name="visual">
+        <pose>0 0 0 0 0 0</pose>
+        <geometry>
+            <mesh filename="package://cob_gazebo_worlds/Media/models/izs-campus/meshes/map.stl" />
```

Comment:
the mesh file still has 0 Byte (empty file)

Reply:
Very strange, in my repo its not empty (8.9MB). I can also view it here: https://github.com/ipa-fog/cob_simulation/blob/feature/izs-campus/cob_gazebo_worlds/Media/models/izs-campus/meshes/map.stl

Can you access this file as well?

Reply:
yes, I can see it there...but honestly, 8.9MB is waaay to large anyway....it will kill gazebo...

Reply:
as it is a big binary file it should at least be tracked via git-lfs and not be included in the git history to not blow up the repo size. Please check that is not in the history (added and remove) before merging

Reply:
I reduced the size to 6MB. However, git-lfs is not working because I can't push lfs assets to a public fork unless the original repo already has LFS objects. Therefore, feel free to close this PR.